### PR TITLE
Updated Samtools NEWS based on merged PRs

### PR DESCRIPTION
**Pull request by jkbonfield**

### CODE REVIEW

File: NEWS [hunk 1]
```diff
@@ -1,24 +1,87 @@
...
+   coverage depth, if the user adds the -J option to the command
+   line (#1163).
+   Also stopped depth from closing stdout, which triggered test fails
+   in pysam (#1208).
+
  * samtools view now accepts remote URIs for FASTA and FAI files.
    Furthermore, the reference and index file can be provided in a single
    argument, such as
    samtools view -T ftp://x.com/ref.fa##idx##ftp://y.com/index.fa.fai a.cram
-
```
Comment (mp15) [inline NEWS:8]: Were these two features dropped, addressed elsewhere or is it a merge conflict?

**Comment by jkbonfield** [comment]
On Wed, May 13, 2020 at 09:39:11AM -0700, Martin Pollard wrote:
> @mp15 commented on this pull request.
> 
> >   * samtools view now accepts remote URIs for FASTA and FAI files.
>     Furthermore, the reference and index file can be provided in a single
>     argument, such as
>     samtools view -T https://urldefense.proofpoint.com/v2/url?u=ftp-3A__x.com_ref.fa-23-23idx-23-23ftp-3A__y.com_index.fa.fai&d=DwICaQ&c=D7ByGjS34AllFgecYw0iC6Zq7qlm8uclZFI0SqQnqBo&r=wodoR_G062E4YLZ-xu5t6g&m=Cb4TNhtt0-odT9E-rC9nshJHXp7ucDzU-ZMvolC35zk&s=q4ndIddAMubNq7Yr6gsHUm587mHAZ_iEyZjq6FI87dY&e=  a.cram
> -
> 
> Were these two features dropped, addressed elsewhere or is it a merge conflict?

What happened with this comment btw?  Did you remove it?  I looked
this morning and couldn't find it in github.

As far as I'm aware the code went in.  I don't see what you mean by
merge conflict.  I had none when making my NEWS update.

James

-- 
James Bonfield (jkb@sanger.ac.uk)
The Sanger Institute, Hinxton, Cambs, CB10 1SA


-- 
 The Wellcome Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.

**Comment by daviesrob** [comment]
It's actually a 1.10 feature.  Search for "A delimiter '##idx##' can be appended".

**Comment by mp15** [comment]
I deleted it when I realised 30 seconds later that the lines I had thought disappeared had just been moved elsewhere

**Comment by daviesrob** [comment]
Rebased with additional NEWS.  Some shuffling may be in order.  For 1.10 we grouped by sub-command, should we do that again?

**Pull request merged**